src/scss/_mixins.scss

```diff
@@ -0,0 +1,256 @@
+/// Output editorial styles for body text.
+/// @example
+///     p {
+///     	@include oEditorialTypographyBody();
+///     }
+@mixin oEditorialTypographyBody() {
+	@include oTypographySerif($scale: (default: 1, XL: 2), $line-height: 1.6);
+	color: oColorsGetColorFor('body', 'text');
+	margin: 0; // undo browser defaults
+}
+
+/// Output editorial styles for lists.
+/// Styles child `li` elements. Apply to a
+/// containing list element such as `ul` or `ol`.
+/// Does not output font styles, these are
+/// inherited (@see oEditorialTypographyBody).
+///
+/// @example Output the styles for an unordered list.
+///     .my-unordered-list {
+///     	@include oEditorialTypographyList('unordered');
+///     }
+///
+/// @example Output the styles for an ordered and unordered list, sharing base list styles.
+///     .my-list {
+///     	@include oEditorialTypographyList();
+///     }
+///
+///     .my-list--ordered {
+///     	@include oEditorialTypographyList('ordered', $include-base-styles: false);
+///     }
+///
+///     .my-list--unordered {
+///     	@include oEditorialTypographyList('unordered', $include-base-styles: false);
+///     }
+///
+/// @param {String|Null} $type [null] - "ordered", "unordered", or null for just the base styles shared by all lists
+/// @param {Boolean} $include-base-styles [true] - set to false to exclude base styles which are shared by all list types
+@mixin oEditorialTypographyList($type: null, $include-base-styles: true) {
+	// Undo browser defaults.
+	@if($include-base-styles) {
+		margin: 0;
+		padding: 0;
+		list-style: none;
+	}
+	// Reset number counter for new ordered list.
+	@if($type == 'ordered') {
+		counter-reset: item;
+	}
+	> li {
+		// Undo browser defaults.
+		@if($include-base-styles) {
+			margin: 0;
+		}
+		&:before {
+			// Allow space for 2-3 numbers for both ordered and unordered lists,
+			// so content aligns between both list types. As an inline pseudo
+			// element a longer count will push list content rather than overlap.
+			@if($include-base-styles) {
+				display: inline-block;
+				box-sizing: border-box;
+				min-width: 3ex;
+				padding-right: oSpacingByName('s1');
+			}
+
+			@if($type == 'unordered') {
+				content: '\2022'; // dot character
+				color: inherit;
+				transform: scale(1.778); // 32px dot character given a parent font-size of 18px
```

Comment:
Are we able to use `calc` here as a progressive enhancement and fallback to this `scale(1.778)` value? I think with `calc` it would mean when someone changes the font-size in their browser it would also change the "dot character" used in our unordered lists.

Reply:
The transform is supported with prefixes back to IE10, and will increase according to font size which is inherited by the list. So I think your concerns here are covered 😊  If I had used `font-size` instead I couldn't use `ex` and `ch` units sensibly.